core/web/evm_transactions_controller.go

```diff
@@ -47,3 +60,131 @@ func (tc *TransactionsController) Show(c *gin.Context) {
 
 	jsonAPIResponse(c, presenters.NewEthTxResourceFromAttempt(*ethTxAttempt), "transaction")
 }
+
+type EvmTransactionController struct {
+	Enabled     bool
+	Logger      logger.SugaredLogger
+	TxmStorage  txmgr.EvmTxStore
```

Comment:
nit: lets name it as TxmStore. The previous name TxmStorageService was an old one, which we replaced with TxmStore, but I see there are still some places using the old name.